cockpit/src/main/res/layout/cockpit_include_activity_layout.xml

```diff
@@ -1,28 +1,36 @@
 <?xml version="1.0" encoding="utf-8"?>
-<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
-            android:layout_width="match_parent"
-            android:layout_height="match_parent">
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
```

Comment:
Orientation parameter in RelativeLayout does nothing.

Reply:
Good point, must've left it out from linear layout.